### PR TITLE
Fix display of dropdown on case studies

### LINKED ISSUE #22221: Version, Language drop-downs non-functional on Case Study pages
Expected behavior: https://v1-18.docs.kubernetes.io/case-studies/spotify/ (drop-downs work)
Actual behavior: https://kubernetes.io/case-studies/spotify/ (drop-downs do not work)

This may be a `z-index` issue or a result of non-translated pages not showing up in autogenerated navs. I'm unsure.

### PR DESCRIPTION
**Pull request by celestehorgan**
Fixes #22221. 

Big thanks to @shuuji3 for the solution to this one! :)

### CODE REVIEW

**Comment by sftim** [comment]
@celestehorgan / @shuuji3 how should a reviewer test this fix?

**Comment by celestehorgan** [comment]
@sftim:

1. Navigate to https://kubernetes.io/case-studies/spotify/ and test the two dropdowns in the menu
2. Compare to https://deploy-preview-22277--kubernetes-io-master-staging.netlify.app/case-studies/spotify/

**Comment by sftim** [comment]
Thanks @celestehorgan 
/lgtm

File: assets/scss/_custom.scss [hunk 1]
```diff
@@ -108,6 +108,10 @@ section {
 
       &.dropdown:hover {
         color: $medium-grey;
+
+        .dropdown-menu {
```
Comment (kbhawkey) [inline assets/scss/_custom.scss:112]: Hi @celestehorgan . This change looks good for adding "hovering" to the two drop down menus.
Currently, you have to "click" the version and language drop down menus. This seems to work for all types, except
the individual case studies pages (spotify, ...).
The case studies home page drop down menus work for me. Not sure why the individual case studies pages are special.

To confirm, this change adds hover to the two drop down menus off the main navbar for all pages.
There is something different about the individual case studies pages.

**Comment by celestehorgan** [comment]
@kbhawkey as usual, thank you for your review and pushing me to ever-greater perfection ;)

The root issue was not including the `scripts.html` partial on the layout template for case study pages – Bootstrap dropdowns have a dependent JS file. 

In the process of solving this I decided to use [Bootstrap's menu alignment classes](https://getbootstrap.com/docs/4.0/components/dropdowns/#menu-alignment) rather than the SCSS in use before. This entailed copying the language & version drop-downs over from Docsy to customize them. 

One final note: As I went through the process of figuring this out, I inadvertently discovered that... we might just be able to delete the case study single.html templates altogether? I can open a very hack-y PR for this if we want to discuss.

File: assets/scss/_custom.scss (hunk 1 above)
Comment (celestehorgan) [reply assets/scss/_custom.scss]: Addressed in the latest commits :)

**Comment by zacharysarah** [comment]
/approve

Adding an explicit hold for review from another chair or tech lead:
/hold

**Comment by tengqm** [comment]
> In the process of solving this I decided to use [Bootstrap's menu alignment classes](https://getbootstrap.com/docs/4.0/components/dropdowns/#menu-alignment) rather than the SCSS in use before. 

+100

File: layouts/partials/navbar-version-selector.html [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-  {{ T "version_menu" }}
+	{{ .Site.Params.version_menu }}
```
Comment (celestehorgan) [inline layouts/partials/navbar-version-selector.html:2]: Note for tomorrow Celeste: change this back to the T string

**Comment by tengqm** [comment]
Pleases squash the commits then it's ready.

**Comment by sftim** [comment]
These commits are logically separate, I'm happy to see this land without squashing further.

**Comment by celestehorgan** [comment]
🤔 I wonder if i can do this

/remove hold

**Pull request merged**